src/contracts/implementations/arbitrable/Arbitrable.js

```diff
@@ -28,12 +29,17 @@ class Arbitrable extends ContractImplementation {
     if (this.metaEvidenceCache[this.contractAddress])
       return this.metaEvidenceCache[this.contractAddress]
 
+    if (this.arbitrableTransactionId === null)
```

Comment:
Can you name this `metaEvidenceID` (not all arbitrable contracts use transactionId as the metaEvidenceID)?

Reply:
Also pass it as a param instead of using this.arbitrableTransactionID and have it default to 0